src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,78 @@
+package duke;
+
+import java.io.IOException;
+
+import command.Command;
+import exception.DukeException;
+import ui.Ui;
+
+/**
+ * Represents a <code>Duke</code> object which is a ChatBot that can keep track of several kinds of tasks.
+ */
+public class Duke {
+    private static final String DATA_DIRECTORY = "data";
+    private static final String SAVED_FILE_PATH = "data/duke.txt";
```

Comment:
I liked the use of static final here to give these strings a named constant for readability!

Reply:
Thanks!